cmd/controller/main.go

```diff
@@ -71,6 +72,7 @@ func main() {
 		PRImage:                  *prImage,
 		ImageDigestExporterImage: *imageDigestExporterImage,
 	}
+	os.Setenv("GORACE", "halt_on_error=1")
```

Comment:
Should this be here ? :thinking:

Reply:
doh, that was a leftover of my blind hacking trying to figure out why baseBuildOverrides was doing nothing yesterday :)